tests/Feature/Jobs/WorkflowRunJobTest.php

```
@@ -55,4 +58,87 @@ public function test_run_job_failed_sets_status_to_failed_and_logs(): void
+            ],
+            id: 'node-1',
+        );
+
+        $job = new RunJob($run, $this->user, $workflow, $executionTree, ['inputs' => []]);
+        $job->handle();
+
+        $run->refresh();
+
+        $this->assertEquals(WorkflowRunStatus::COMPLETED, $run->status);
+        $this->assertStringContainsString('command-output-line', $run->getLogContent());
+    }
```

PHPMD (2.15.0)

[error] The method test_verbose_run_logs_command_output is not named in camelCase. (undefined)

(CamelCaseMethodName)